src/eterna/folding/__tests__/Folder.test.ts

```diff
@@ -31,14 +31,14 @@ test('NuPACK:suboptstructuresNoOligos', () => {
      .then((folder) => {
         if (folder === null) return;
 
-        let sequence = Sequence.fromSequenceString("auauauagaaaauauaua")
+        let sequence = Sequence.fromSequenceString("GGGAACGACUCGAGUAGAGUCGAAAAGAUAUAGAAAGCGCGACUAUAUCAAUAAACGCGCAAAAUAAAUAAAUAAAGAUCAGUUUCGACUGAUCAAAAGAAACAACAACAACAAC")
```

Comment:
It looks like you've modified existing tests, but we still want to test that use case! Please add a new test for the pkots behavior